### DISCUSSION TITLE
can i use rustdesk as a commercial remote desktop

### DISCUSSION THREAD

**Discussion by yashpatel1598** [rustdesk/rustdesk#3236] [General]
hi rustdesk team,
i want to know that can i use restdusk as a commercial remote desktop in my in house factory in which i will change some lines of code and also change name from rustdesk to my organization name so cloud i do it or not or is there any other way can i do this @rustdesk @Kingtous @Heap-Hop @fufesou

**Comment by ion-tichy** [comment]
Yes you should be able to do so since the code is under AGPL.

Note that this also means that you have to release the sources of  your modified version under AGPL aswell.
